platformio.ini

```diff
@@ -10,23 +10,25 @@
 
 [common]
 board_f_cpu = 160000000L
-lib_deps_builtin = 
+lib_deps_builtin =
   SPI
-lib_deps_external = 
+lib_deps_external =
   RF24
   WiFiManager
   ArduinoJson
-build_flags = !echo -D MILIGHT_HUB_VERSION=\\\"$(git describe --always)\\\"
+build_flags = !echo -D MILIGHT_HUB_VERSION="\"$(git describe --always)\""
+; build_flags = -D MILIGHT_HUB_VERSION=\"1.1.0\" -D MILIGHT_UDP_DEBUG
 # -D MILIGHT_UDP_DEBUG
+upload_speed = 9600
 
-[env:nodemcuv2]
-platform = espressif8266
-framework = arduino
-board = nodemcuv2
-lib_deps =
-  ${common.lib_deps_builtin}
-  ${common.lib_deps_external}
-build_flags = ${common.build_flags} -D FIRMWARE_VARIANT=\\\"nodemcuv2\\\"
+; [env:nodemcuv2]
```

Comment:
Can we uncomment this? If you want to build for a specific environment, you can do `platformio run -e d1_mini`.